### DISCUSSION TITLE
Autobet for Blackjack in GTA 5 Online

### DISCUSSION THREAD

**Discussion by ShyRoxxx** [MarkusJx/autobet#64] [Ideas]
Hey, is it possible that you might create an autobet for blackjack using this data? https://www.coolcat-casino.com/images/uploads/2010/11/blackjack-cheat-sheet.jpg

**Comment by MarkusJx** [comment]
Yes, that would be possible.

But I won't do that. Let me explain:
According to [this](https://www.onlinegambling.com/blackjack/odds/), your chance of winning is ~42%. The chance of a push is ~8%, the chance of losing is ~49%. So basically, you'd lose money or just make such small amounts, making this feature completely useless.

Example (probably not *that* correct, but I'm trying to prove a point here):
If you take $100, and spend $1 in each game, you could play 100 games and you'd lose 49 of them, losing you $49.
So you'd have $51 left. You'd also push on 8 games, leaving you with $51, as you didn't lose but also didn't make any money.
Now there are 41 Games you'd win, ~5% of those will be a black jack. If I recall correctly, you'll get 3 times the money you've placed back ($3, leaving you with $2 of winnings).
With these 5 games where you get a black jack you'd have $61.
There are now 36 Games left, where you'd win, making you $36, which calculates to a grand total of $97 after spending $100.
So you'd lose $3. While that not being much money, this would be immensely useless as it would just lose money without any benefits. If you just want to lose money, I'd suggest playing the game, this will lose you heaps of cash.

And all of this makes sense, if you think about how casinos work, as a casino's job is not to lose money, they want to make money. And of course that works pretty well if you look at cities like Las Vegas. So it makes a lot of sense that playing blackjack will lose you money (assuming blackjack in-Game works like real blackjack).

In Summary, yes, that would be *technically* possible, but from a logical standpoint it makes no sense at all and should not exist.
The only reason why making money through the horse racing is possible, is because the implementation is broken, and R* already nerved this with numerous updates. But it still works.
